staging/src/k8s.io/apiserver/pkg/server/genericapiserver_graceful_termination_test.go

```diff
@@ -200,10 +205,12 @@ func TestGracefulTerminationWithKeepListeningDuringGracefulTerminationDisabled(t
 	}, nil)
 
 	// start the API server
-	stopCh, runCompletedCh := make(chan struct{}), make(chan struct{})
+	runCompletedCh := make(chan struct{})
```

Comment:
I don't like moving the stopCh replacement away. It is crucial in this test. So better create another ctx+cancel here.

Reply:
The rationale was that ctx is more like a per-test than per-apiserver context, but I can also leave the setup here.